src/test/sock_tests.cpp

```diff
@@ -38,7 +38,6 @@ BOOST_AUTO_TEST_CASE(constructor_and_destructor)
 {
     const SOCKET s = CreateSocket();
     Sock* sock = new Sock(s);
-    BOOST_CHECK_EQUAL(sock->Get(), s);
```

Comment:
Checking that you actually got the socket you wanted seems valuable (particularly for move operations). Maybe:

```c++
class Sock {
public:
    /** For unit tests */
    bool CheckSocket(SOCKET s) const { return s == m_socket; }
...
};

//  BOOST_CHECK_EQUAL(sock->Get(), s);
    BOOST_CHECK(sock->CheckSocket(s));
```

Reply:
Done, but named it `operator==` instead of `CheckSocket`.

Reply:
Could consider updating the commit message for `net: remove now unnecessary Sock::Get()`

Reply:
Done.